oshc/main/static/main/js/script.js

```diff
@@ -5,4 +5,15 @@ $(document).ready(function() {
       scrollTop: 0
     }, 1000);
   });
+
+  $(".dropdown").hover(
```

Comment:
since we are using `bootstrap.min.js`, I think we should use the default drop-down menu of Bootstrap. No need to write JS code for that.